crucible-llvm/src/Lang/Crucible/LLVM/Globals.hs

```diff
@@ -133,8 +134,7 @@ makeGlobalMap ctx m = foldl' addAliases globalMap (Map.toList (llvmGlobalAliases
            ++ ": "
            ++ err)
      where showSymbol sym =
-             show $ let ?config = LPP.Config False False False
-                    in LPP.ppSymbol $ sym
+             show $ ppLLVMLatest $ LPP.ppSymbol $ sym
```

Comment:
I opted to change the `Config` used here for consistency's sake. The `ppSymbol` function doesn't make use of any `Config`-specific options anyway.